YoutubeDownloader.Core/Tagging/MediaTagInjector.cs

```diff
@@ -60,7 +60,7 @@ private async Task InjectMusicMetadataAsync(
             mediaFile.SetAlbum(recording.Album);
     }
 
-    private async Task InjectThumbnailAsync(
+    private static async Task InjectThumbnailAsync(
```

Comment:
```suggestion
    private async Task InjectThumbnailAsync(
```

Please avoid making changes unrelated to the main goal of this PR